tests/unit/customizations/configure/test_importer.py

```diff
@@ -93,6 +93,31 @@ def test_import_downloaded_bad_headers(self):
         with self.assertRaises(CredentialParserError):
             self.import_command(args=['--csv', content], parsed_globals=None)
 
+    def test_raises_error_when_plain_file_path_passed(self):
+        with open('temp_creds.csv', 'w') as f:
+            f.write('User name,Access key ID,Secret access key\nuser,AKID,SAK')
+        try:
+            with self.assertRaises(ValueError) as cm:
+                self.import_command(args=['--csv', 'temp_creds.csv'], parsed_globals=None)
+            self.assertIn("file://", str(cm.exception))
```

Comment:
I think it would be a bit safer to assert that some other unique part of the error message is present because `file://` does occur twice in the message.